packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx

```diff
@@ -1,12 +1,33 @@
-import * as React from 'react';
 import { ColorPicker } from 'office-ui-fabric-react/lib/ColorPicker';
+import * as React from 'react';
 
 export interface IBasicColorPickerExampleState {
   color: string;
 }
 
 export class ColorPickerBasicExample extends React.Component<any, IBasicColorPickerExampleState> {
+  constructor(props: any) {
+    super(props);
+
+    this.state = {
+      color: '#ffffff'
+    };
+
+    this._updateColor = this._updateColor.bind(this);
+  }
+
   public render(): JSX.Element {
-    return <ColorPicker color="#FFFFFF" />;
+    return (
+      <div style={{ display: 'flex' }}>
+        <ColorPicker color={this.state.color} onColorChanged={this._updateColor} />
+        <div style={{ backgroundColor: this.state.color, width: 100, height: 100, margin: 16, border: '1px solid #c8c6c4' }} />

```

Comment:
Since this is an example, maybe we can encourage the use of mergeStyles here instead of inline styles?

Reply:
Is there an easy way to apply mergeStyles to a div without having to create interfaces, a `*.base.tsx` file, and so on? I can add that, it just feels like overkill for a simple component.

Reply:
This is good feedback on our examples, as I can't find any that use mergeStyles and this could be an opportunity to demonstrate it. But I think it's out of scope for this PR given that it matches what we do in other examples.